main/main.c

```diff
@@ -496,6 +497,21 @@ static void batchMakeTags (cookedArgs *args, void *user CTAGS_ATTR_UNUSED)
 #ifdef HAVE_JANSSON
 void interactiveLoop (cookedArgs *args CTAGS_ATTR_UNUSED, void *user CTAGS_ATTR_UNUSED)
 {
+#ifdef HAVE_SECCOMP
+	if (Option.secure) {
+		/* As of jansson 2.6, the object hashing is seeded off
+		   of /dev/urandomm, so trigger the hash seeding
+		   before installing the syscall filter.
+		*/
+		json_t * tmp = json_object ();
+		json_decref (tmp);
+
+		if (installSyscallFilter ()) {
+			error (FATAL, "install_syscall_filter failed");
```

Comment:
exit(1) may be needed here. In interactive mode, I'm afraid 'error(FATAL' doesn't make the ctags process exit.  I will fix this. I have to think about the behavior of  jsonErrorPrinter in early stage.